models/model_choice.py

```diff
@@ -95,20 +95,6 @@ def net(net_params, num_channels, inference=False):
 
             model = LayersEnsemble(model, conc_point=conc_point)
 
```

Comment:
Was RGBN Baseline removed entirely?

Reply:
yeah, should I put it ? like as a option?